packages/fonts/src/get-font-faces.ts

```diff
@@ -9,13 +9,31 @@ export type PartialFontAsset = {
 
 export type FontFace = {
   fontFamily: string;
-  fontStyle: FontMeta["style"];
-  fontWeight: number;
   fontDisplay: "swap" | "auto" | "block" | "fallback" | "optional";
   src: string;
+  fontStyle?: FontMetaStatic["style"];
+  fontWeight?: number | string;
+  fontStretch?: string;
 };
 
-const formatFace = (asset: PartialFontAsset, format: string) => {
+const formatFace = (asset: PartialFontAsset, format: string): FontFace => {
+  if ("variationAxes" in asset.meta) {
+    const { wght, wdth } = asset.meta?.variationAxes ?? {};
+    const fontStretch = wdth
+      ? { fontStretch: `${wdth.min}% ${wdth.max}%` }
+      : undefined;
+    const fontWeight = wght
+      ? { fontWeight: `${wght.min} ${wght.max}` }
+      : undefined;
```

Comment:
Seems like `in` vs `undefined` a little complicates the logic. Maybe stick to always undefined?

Reply:
we don't want the property to be in there defined at all, unless we provide a default

Reply:
I could set fontWeight: normal by default, but is it worth specifying meaningless default and publishing it to all sites instead of having this?

Reply:
oh you meant that its only there in the data structure but not rendered in the end, hmm, you are probably right

Reply:
fixed